### PR TITLE
refactor: remove access token persist storage

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

remove keychain storage for access token per discussion

## Testing

- [x] sign in
- [x] get access token for user info
- [x] get access token for resource
- [x] re-open app, get access token for user info
- [x] re-open app, get access token for resource

## Reviewers

@logto-io/eng

### CODE REVIEW

File: Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ struct ContentView: View {
 
     init() {
         guard let config = try? LogtoConfig(
-            endpoint: "https://logto.dev",
+            endpoint: "http://localhost:3001",
```
Comment (demonzoo) [inline Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift:21]: Please revert this change as it's for local test

**Review by demonzoo — APPROVED** [review]

File: Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift (hunk 1 above)
Comment (gao-sun) [reply Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift:21]: sure. dkw logto.dev still have the resource issue. will figure out tmw.

Comment (demonzoo) [reply Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift:21]: 👌

**Review by wangsijie — APPROVED** [review]
